scripts/adjust.js

```diff
@@ -42,7 +42,25 @@ const addSlateCoverImage = db.schema.table("slates", function (table) {
   table.jsonb("coverImage").nullable();
 });
 
-Promise.all([addSlateCoverImage]);
+const addSettingsColumnToUsersTable = db.schema
+  .table("users", (table) => {
+    table.jsonb("settings").defaultTo("{}");
+  })
+  .then(() =>
+    db("users").update(
+      "settings",
+      db.raw("jsonb_build_object('filecoin', users.data -> 'settings' )")
+    )
+  );
+
```

Comment:
Also don't forget to add the createSurveysTable function to the adjust script so that I can run it for other files: 
```
const dropSurveysTable = db.schema.dropTableIfExists("surveys");

const createSurveysTable = createTableIfNotExists("surveys", function (table) {
  table.uuid("id").primary().unique().notNullable().defaultTo(db.raw("uuid_generate_v4()"));
  table.uuid("ownerId").references("id").inTable("users");

  // What do you currently use for saving things on the web?
  table.boolean("prevToolsDropbox").defaultTo(false);
  table.boolean("prevToolsArena").defaultTo(false);
  table.boolean("prevToolsPinterest").defaultTo(false);
  table.boolean("prevToolsGoogleDrive").defaultTo(false);
  table.boolean("prevToolsBrowserBookmarks").defaultTo(false);
  table.string("prevToolsOther").defaultTo(null);

  // What are you interested in using Slate for?
  table.boolean("useCasesPersonalStorage").defaultTo(false);
  table.boolean("useCasesPublicFileSharing").defaultTo(false);
  table.boolean("useCasesArchiving").defaultTo(false);
  table.boolean("useCasesBookmarking").defaultTo(false);
  table.boolean("useCasesMoodboarding").defaultTo(false);
  table.string("useCasesOther").defaultTo(null);

  // How did you find out about Slate?
  table.boolean("referralFriend").defaultTo(false);
  table.boolean("referralTwitter").defaultTo(false);
  table.boolean("referralIpfsFilecoinCommunity").defaultTo(false);
  table.string("referralOther").defaultTo(null);
});
```